GoogleSignIn/Tests/Unit/GIDVerifiableAccountDetailTest.m

```diff
@@ -0,0 +1,23 @@
+#import <XCTest/XCTest.h>
+
+#import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h"
+
+@interface GIDVerifiableAccountDetailTests : XCTestCase
+@end
+
+@implementation GIDVerifiableAccountDetailTests
```

Comment:
I think it would be worth testing the edge case that the provided account detail yields a `nil` value when asking for the `scope`.

Reply:
Done.